webpack/assets/javascripts/react_app/components/notifications/notifications.fixtures.js

```diff
@@ -83,11 +83,11 @@ export const stateWithUnreadNotifications = immutable({
   },
 });
 
-export const serverResponse = `{ "notifications":[
+export const serverResponse = `{"data": { "notifications":[
```

Comment:
why? I don't assume the server response changed?

Reply:
axios returns a new object, the body response encapsulates in data. 
I've extracted this before, though @sharvit  thinks it better to keep the whole axios response object for later use.